### ISSUE TITLE
Only the first character appears for synonyms for SARS-CoV-2 

### ISSUE THREAD

**Issue by justaddcoffee** [Knowledge-Graph-Hub/kg-covid-19#232]
### Describe the bug

Only the first character appears for synonyms for SARS-CoV-2, for example

### To Reproduce

http://kg-hub-rdf.berkeleybop.io/blazegraph/#query
```
  prefix owl: <http://www.w3.org/2002/07/owl#>
  prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
  prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
  prefix bl: <https://w3id.org/biolink/vocab/>
  prefix ex: <https://www.example.org/UNKNOWN/>
  SELECT ?protein ?p ?v WHERE
    {
    ?protein bl:category bl:Protein; 
             ex:ncbi_taxid ?ncbitaxonid;
             ?p ?v .
     FILTER NOT EXISTS { 
      ?v bl:category bl:Publication . 
      ?v ex:provided_by ?scibitecord  
      . FILTER(?scibitecord="SciBite-CORD-19")}
     FILTER(?ncbitaxonid="2697049")
    }
```

Notice synonyms are always 1 character: 

8 | <http://identifiers.org/uniprot/P0DTD3> | bl:synonym | "O"^^<http://www.w3.org/2001/XMLSchema#string>
-- | -- | -- | --

## Expected behavior

Should see the full string for synonyms

**Comment by justaddcoffee** [comment]
should be closed by #231

**Comment by justaddcoffee** [comment]
Fixed, see for example this query:

```
prefix owl: <http://www.w3.org/2002/07/owl#>
prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
prefix bl: <https://w3id.org/biolink/vocab/>
SELECT ?protein ?p ?v WHERE 
{
<http://identifiers.org/uniprot/P0DTD1> bl:category bl:Protein; <https://www.example.org/UNKNOWN/ncbi_taxid> ?ncbitaxonid;
 ?p ?v .
 FILTER(?ncbitaxonid="2697049")
}
```

which yields these (multicharacter!) synonyms:

7 |   | bl:synonym | "1a-1b"^^<http://www.w3.org/2001/XMLSchema#string>
-- | -- | -- | --
8 |   | bl:synonym | "rep"^^<http://www.w3.org/2001/XMLSchema#string>
